open-metadata-implementation/user-interfaces/presentation-server/ssl/keys/server.cert

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
+MIIDnjCCAoYCCQDR7DKQq8tpSzANBgkqhkiG9w0BAQsFADCBkDESMBAGA1UECAwJ
```

Comment:
see comment above about certs - if we ship one may be useful to doc how. but can be later PR..

Reply:
as above let's handle in #1403

Reply:
Agreed. Issue addressed in #1403